configure.ac

```diff
@@ -1,4 +1,5 @@
 AC_INIT([testPkg], 1.0)
+AC_CONFIG_HEADER([src/config.h])
```

Comment:
This should really have been a fixup to the previous commit introducing `src/config.h.in` (which on its own doesn't do anything).